deploy-manage/users-roles/cluster-or-deployment-auth/managed-credentials-eck.md

```diff
@@ -66,13 +66,17 @@ If you are using the `elastic` user credentials in your own applications, they w
 To regenerate all auto-generated credentials in a namespace, run the following command:
 
 ```sh
-kubectl delete secret -l eck.k8s.elastic.co/credentials=true
+kubectl delete secret -l eck.k8s.elastic.co/credentials=true,common.k8s.elastic.co/type!=kibana
 ```
 
 ::::{warning}
 This command regenerates auto-generated credentials of **all** {{stack}} applications in the namespace.
 ::::
 
+:::{note}
+When deleting secrets so they can be regenerated, make sure to exclude {{kib}} secrets by specifying `type!=kibana`. {{kib}} secrets contain encryption keys, which should not be deleted.
```

Comment:
We should qualify this statement to be true up to version 3.2. 

@shainaraskas do you have a recommendation how we should best include instructions that only pertain to older versions of the software, without cluttering the whole docs page with exceptions?

Reply:
I'd suggest: 

* use an [applies switch](https://elastic.github.io/docs-builder/syntax/applies-switch/) and put your code snippets in there. one tab for 3.0 and one for 3.2 - put your 3.2 tab first.

* in your note, just preface it with 'In ECK versions 3.1 and earlier ...' 'In ECK versions prior to 3.2 ...'